docs/publish/custom-protocol-handler.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Trust custom protocol handlers that launch add-ins
+description: How to use group policies for protocol handler trust in the registry to launch add-ins.
+ms.topic: how-to
+ms.date: 08/05/2024
+ms.localizationpriority: medium
+---
+
+# Trust custom protocol handlers to launch add-ins
+
+Protocol handlers are registered with the operating system to allow an app to be launched from a URI (for example, how `mailto:` launchs an email client). Add-ins can also be launched from protocol handlers. This article explains how to automatically trust these custom protocol handlers by using admin group policies.
+
+Every add-in and protocol pair needs to be trusted. This either comes in the form end-user consent or admin group policies. Similarly, admins can block certain add-in and protocol pairs.
+
+> [!IMPORTANT]
+> This information about trusting custom protocol handlers that launch add-ins through the registry only pertains to Windows.
+
+## Registry key format
+
+To automatically trust a custom protocol handler that launches an add-in, create a registry key at one of the following locations. Note that `<add-in id>` refers to the [Id element](/javascript/api/manifest/id) specified in your add-in's manifest.
```

Comment:
If this feature works with add-ins using the unified manifest, then we should mention where the ID is found in that manifest.

```suggestion
To automatically trust a custom protocol handler that launches an add-in, create a registry key at one of the following locations. Note that `<add-in id>` refers to the [Id element](/javascript/api/manifest/id) specified in the add-in only manifest or to the "id" property of the unified manifest..
```